File: SofiForce.Models/Models/SearchModels/OperationRequestDetailSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class OperationRequestDetailSearchModel : BaseSearchModel
    {

		
		public Int32? ClientId { get; set; }
		public int? OperationTypeId { get; set; }
		public int? RepresentativeId { get; set; }
		public Int32? OperationId { get; set; }
		public Int32? ClientTypeId { get; set; }
		public Int32? RegionId { get; set; }
		public Int32? GovernerateId { get; set; }
		public Int32? CityId { get; set; }
		public Int32? LocationLevelId { get; set; }
		public Int32? OperationStatusId { get; set; }
		public Int32? OperationRejectReasonId { get; set; }
		public DateTime? OperationDate { get; set; }


	}
}

File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/OperationRequestDetailSearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface OperationRequestDetailSearchModel extends BaseSearchModel {
    clientId: number | null;
    operationId: number | null;
    clientTypeId: number | null;
    regionId: number | null;
    governerateId: number | null;
    cityId: number | null;
    locationLevelId: number | null;
    operationStatusId: number | null;
    operationRejectReasonId: number | null;

    operationDate: Date | null;
}